<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>731</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="statusTip">
      <string>离线视频运行</string>
     </property>
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="run_offline_button">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>190</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>运行</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="filePath">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>431</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="FilePath_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>视频路径：</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="result_text">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>440</y>
        <width>501</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="stop_offline_button">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>230</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>结束</string>
      </property>
     </widget>
     <widget class="QLabel" name="result_video">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>491</width>
        <height>321</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="statusTip">
      <string>在线运行</string>
     </property>
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>754</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_file"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionsvs">
   <property name="text">
    <string>svs</string>
   </property>
  </action>
  <action name="actionMenu">
   <property name="text">
    <string>Menu</string>
   </property>
  </action>
  <action name="actionNew_file">
   <property name="text">
    <string>New file</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
